src/settings-ui/Settings.UI/Views/GeneralPage.xaml

```diff
@@ -3,394 +3,331 @@
+        <converters:BoolToVisibilityConverter x:Key="VisibleIfTrueConverter" />
+        <converters:BoolNegationConverter x:Key="NegationConverter" />
         <localConverters:UpdateStateToBoolConverter x:Key="UpdateStateToBoolConverter" />
-        <localConverters:StringToInfoBarSeverityConverter x:Key="StringToInfoBarSeverityConverter"/>
+        <localConverters:StringToInfoBarSeverityConverter x:Key="StringToInfoBarSeverityConverter" />
     </Page.Resources>
 
-    <controls:SettingsPageControl x:Uid="General"
-                                  ModuleImageSource="ms-appx:///Assets/Modules/PT.png">
+    <controls:SettingsPageControl x:Uid="General" ModuleImageSource="ms-appx:///Assets/Modules/PT.png">
         <controls:SettingsPageControl.ModuleContent>
             <StackPanel Orientation="Vertical">
                 <controls:SettingsGroup x:Uid="General_Version" Margin="0,-32,0,0">
-                    <controls:Setting Header="{Binding PowerToysVersion}" Icon="&#xE895;">
-                        <controls:Setting.Description>
+                    <labs:SettingsCard Header="{Binding PowerToysVersion}" HeaderIcon="{ui:FontIcon Glyph=&#xE895;}">
+                        <labs:SettingsCard.Description>
                             <StackPanel Orientation="Vertical">
-
                                 <TextBlock Style="{StaticResource SecondaryTextStyle}">
-                                <Run x:Uid="General_VersionLastChecked" />
-                                <Run Text="{Binding UpdateCheckedDate, Mode=OneWay}"/>
+                                    <Run x:Uid="General_VersionLastChecked" />
+                                    <Run Text="{Binding UpdateCheckedDate, Mode=OneWay}" />
                                 </TextBlock>
-                                <HyperlinkButton x:Uid="ReleaseNotes"
-                                                 NavigateUri="https://github.com/microsoft/PowerToys/releases/"
-                                                 Margin="0,2,0,0"/>
+                                <HyperlinkButton
+                                    x:Uid="ReleaseNotes"
+                                    Margin="0,2,0,0"
+                                    FontWeight="SemiBold"
+                                    NavigateUri="https://github.com/microsoft/PowerToys/releases/" />
+                            </StackPanel>
+                        </labs:SettingsCard.Description>
+                        <Grid Visibility="{Binding PowerToysUpdatingState, Mode=OneWay, Converter={StaticResource UpdateStateToBoolConverter}, ConverterParameter=UpToDate}">
+                            <StackPanel
+                                VerticalAlignment="Center"
+                                Orientation="Horizontal"
+                                Spacing="18"
+                                Visibility="{Binding Mode=OneWay, Path=IsNewVersionDownloading, Converter={StaticResource VisibleIfTrueConverter}}">
```

Comment:
We can use a (more) standard BoolToVisibilityConverter

Reply:
Fixed!